deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml

```diff
@@ -26,10 +26,10 @@ spec:
               mountPath: /data
           resources:
             limits:
-              cpu: "2"
+              cpu: "4"
```

Comment:
Let's bump these to 7. dhstore runs on `r5n.2xlarge` and those nodes have 8 CPUs.